internal/storage/v2/clickhouse/tracestore/config.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package tracestore
+
+import (
+	"time"
+
+	"github.com/asaskevich/govalidator"
+)
+
+type Config struct {
+	// Protocol is the protocol to use to connect to ClickHouse.
+	// Supported values are "native" and "http". Default is "native".
+	Protocol string `mapstructure:"protocol" valid:"in(native|http),optional"`
+	// Addresses contains a list of ClickHouse server addresses to connect to.
+	Addresses []string `mapstructure:"addresses" valid:"required"`
+	// Auth contains the authentication configuration to connect to ClickHouse.
+	Auth AuthConfig `mapstructure:"auth"`
+	// DialTimeout is the timeout for establishing a connection to ClickHouse.
+	DialTimeout time.Duration `mapstructure:"dial_timeout"`
```

Comment:
could we use https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/confighttp ?

Reply:
There are some other HTTP values in the Clickhouse.Options struct - I can add those in a follow-up PR